datafusion/core/src/datasource/physical_plan/parquet/row_groups.rs

```diff
@@ -923,6 +944,7 @@ mod tests {
         let metrics = parquet_file_metrics();
         assert_eq!(
             prune_row_groups_by_statistics(
+                &schema,
```

Comment:
Looks like these are existing tests? Should we add one test of `prune_row_groups_by_statistics` which `schema` (file schema) is different to predicate schema?

Reply:
That is a good idea -- I will do so